setup.py

```diff
@@ -11,7 +11,7 @@
 from setuptools.command.build_py import build_py
 
 DESCRIPTION = "Raiden contracts library and utilities"
-VERSION = "0.37.0-beta"
+VERSION = "0.37.0"
```

Comment:
why is this not `-beta2`?

Reply:
wait....that's exactly what I tried to fix here. give me a second!